helm/jupyterhub-home-nfs/templates/configmap.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-ganesha-config
+data:
+  ganesha.conf: |
```

Comment:
Since now we're specifying ganesha.conf as a configmap, let's remove it from https://github.com/2i2c-org/jupyterhub-home-nfs/blob/main/nfs-ganesha/ganesha.conf and stop copying it in the Dockerfile to avoid confusion.

Reply:
I was keeping the config file to make sure the Dockerfile works on its own but I guess it's fine to remove since we're the only ones using the Dockerfile anyway